lib/std/builtin.zig

```diff
@@ -141,23 +141,49 @@ pub const Mode = enum {
 /// This data structure is used by the Zig language code generation and
 /// therefore must be kept in sync with the compiler implementation.
 pub const CallingConvention = enum {
+    /// This is the default Zig calling convention used when not using `export` on `fn`
+    /// and no other calling convention is specified.
     Unspecified,
+    /// Matches the C ABI for the target.
+    /// This is the default calling convention when using `export` on `fn`
+    /// and no other calling convention is specified.
     C,
+    /// This makes a function not have any function prologue or epilogue,
+    /// making the function uncallable in regular Zig code.
+    /// This can be useful when integrating with assembly.
```

Comment:
Thats not very correct, see https://github.com/ziglang/zig/blob/21d9f0c2a123a0939881ba55bad84e4bae990e47/lib/compiler_rt/count0bits.zig#L142 until line 160.
One can do what a function would do on the target, which makes it callable by regular Zig code.

Reply:
A function with `callconv(.Naked)` itself is not callable in normal Zig code, but you can take the code in the naked function and export it into something else and then call that, like it is done here. I will add the word "itself" to give more attention to this function marked as naked specifically.